lib/rubocop/cop/style/redundant_return.rb

```diff
@@ -115,6 +117,14 @@ def check_begin_node(node)
 
           check_return_node(last_expr)
         end
+
+        def allow_multiple_return_values?
+          cop_config['AllowMultipleReturnValues'] || false
+        end
+
+        def message(_node = nil)
+          allow_multiple_return_values? ? MULTI_RETURN_MSG : MSG
```

Comment:
I think this message should vary not only depending on whether the return value is multiple values and not based on whether this is allowed in general or not. Otherwise the message gets confusing. Right now you'd be getting this for non-multiple values as well.